LayoutTests/TestExpectations

```diff
@@ -2187,6 +2187,15 @@ imported/w3c/web-platform-tests/svg/animations/repeat-event.svg [ Skip ]
 imported/w3c/web-platform-tests/svg/animations/seeking-events-2.html [ Failure Pass ]
 imported/w3c/web-platform-tests/svg/animations/dependent-begin-on-syncbase.html [ Failure Pass ]
 imported/w3c/web-platform-tests/svg/animations/dependent-end-on-syncbase.html [ Failure Pass ]
+imported/w3c/web-platform-tests/svg/path/bearing/absolute.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/bearing/relative.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/bearing/zero.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/closepath/segment-completing.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathlength-circle-mutating.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage.svg [ Skip ]
```

Comment:
Why do we need to skip those?

Reply:
I can make them [ ImageonlyFailure ], I skipped them because we are failing them.

Reply:
I'll let you be the judge of what's best, but skipping them means that we wouldn't be able to determine potential progressions from the bots.

Reply:
True - I will update them post EWS to [ ImageonlyFailure ] run before merging.

Reply:
Changed to `ImageOnlyFailure`